docs/index.rst

```diff
@@ -1,43 +1,54 @@
+.. _package-template:
+
 Welcome to Astropy Package Template's documentation!
 ====================================================
 
-The Astropy package template is designed to help quickly create new Python packages within the Astropy ecosystem.
+The Astropy package template is designed to help quickly create new Python
+packages within the Astropy ecosystem. This package mirrors the layout of the
+main `Astropy_` repository, as well as reusing much of the helper code used to
+organize `Astropy_`.
+
+This documentation will guide you through getting started with the package
+template and updating your package when the template is updated.
 
 Getting Started
 ---------------
 
 The Astropy Package template uses the `Cookiecutter
-<http://cookiecutter.readthedocs.io/>`_ project to make it easier to get
-started. To use the package template you need cookiecutter installed, depending
-on how you have Python installed it can be obtained through either pip or
-conda::
+<http://cookiecutter.readthedocs.io/>`_ project to make it easier to customise
+the template for your package. To use the package template you need cookiecutter
+installed. The package template also optionally makes use of 'gitpython' to
+setup the ``astropy_helpers`` submodule. Depending on how you have Python
+installed these packages can be obtained through either pip or conda::
 
-  pip install cookiecutter
+  pip install cookiecutter gitpython
```

Comment:
I would swap the order of the conda and pip so that conda comes first.  New users usually just do things in order, and if they are on conda it's safer for them to try that first.